src/obsidian_ai_hub/agents/registry.py

```
@@ -376,6 +387,57 @@ def memory_search(
+        except Exception as exc:
+            logger.exception("agent_delegate failed")
+            return json.dumps(
+                {
+                    "status": "failed",
+                    "agent_id": agent_id,
+                    "agent_name": None,
+                    "depth": 1,
+                    "final_answer": None,
+                    "used_tools": [],
+                    "created_hitl_run_ids": [],
+                    "error": f"委譲処理で予期しないエラーが発生しました: {str(exc)}",
```

Ruff (0.16.3)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)